susanchen3/liferay-portal
# LPS-202783 Refactor ReportGeneratedInDefaultPathWhenWriteProtectedDirectory

**Test Plan**
- Create non-root user
- Create write protected directory
- Run upgrade tool as non-root user trying to generate upgrade report in protected directory
- Check location of upgrade report in the logs
- Delete non-root user
- Delete write protected directory


**JIRA Ticket:**
https://liferay.atlassian.net/browse/LPS-202783
